src/webpackPages.js

```diff
@@ -72,16 +73,34 @@ const webpackPages = (globalOptions) => {
     }
 
     const finishAll = () => {
+      if (typeof globalOptions.webpack === 'function') globalOptions.webpack = globalOptions.webpack(globalOptions)
+      if (!outputFiles || Object.keys(outputFiles).length < 1) {
+        rm(path.join(metalsmith._directory, '_tempOutput'), () => { })
+        const webpackError = 'No outputFiles for webpack'
+        console.log(webpackError)
+        if (globalOptions.callback) return globalOptions.callback(new Error(webpackError))
+      }
       globalOptions.webpack.entry = outputFiles
-      metalsmith.webpack = globalOptions.webpack
-      webpack(globalOptions.webpack, err => {
-        if (!globalOptions.webpack.devServer) {
-          rm(path.join(metalsmith._directory, '_tempOutput'), () => { })
+      webpack(globalOptions.webpack, (err, stats) => {
+        if (err) {
+          console.log(err.stack || err)
+          if (err.details) {
+            console.log(err.details)
+          }
```

Comment:
More logging?

Reply:
Also, as this is an error from the webpack process- I think it may be good to keep in for surfacing an error here.